### PR TITLE
`defaults-aws` also nukes the SG's IPv6 egress rule

### LINKED ISSUE #400: `defaults-aws` don't delete IPv6 security group rules.
After running `cloud-nuke defaults-aws`, there is a security group rule with an IPv6 range:
```
"IpPermissionsEgress": [
                {
                    "IpProtocol": "-1",
                    "IpRanges": [],
                    "Ipv6Ranges": [
                        {
                            "CidrIpv6": "::/0"
                        }
                    ],
                    "PrefixListIds": [],
                    "UserIdGroupPairs": []
                }
            ],
```

### PR DESCRIPTION
**Pull request by marinalimeira**
## Description

Fixes #400.

## TODOs

Read the [Gruntwork contribution guidelines](https://gruntwork.notion.site/Gruntwork-Coding-Methodology-02fdcd6e4b004e818553684760bf691e).

- [x] Update the docs.
- [x] Run the relevant tests successfully, including pre-commit checks.
- [x] Ensure any 3rd party code adheres with our [license policy](https://www.notion.so/gruntwork/Gruntwork-licenses-and-open-source-usage-policy-f7dece1f780341c7b69c1763f22b1378) or delete this line if its not applicable.
- [x] Include release notes. If this PR is backward incompatible, include a migration guide.
- [x] Attention Grunts - if this PR adds support for a new resource, ensure the `nuke_sandbox` and `nuke_phxdevops` jobs in `.circleci/config.yml` have been updated with appropriate exclusions (either directly in the job or via the `.circleci/nuke_config.yml` file) to prevent nuking IAM roles, groups, resources, etc that are important for the test accounts.

## Release Notes

Updated `defaults-aws` to nuke IPv6 security group's egress rule

### CODE REVIEW

**Review by Etiene — APPROVED** [review]
The tests failed, but they seem unrelated to these changes, and the code looks good to me! This will probably require another approval though

**Review by ellisonc — APPROVED** [review]

File: aws/ec2.go [hunk 1]
```diff
@@ -729,6 +743,16 @@ func (sg DefaultSecurityGroup) nuke() error {
 				return fmt.Errorf("error deleting eggress rule: %s", errors.WithStackTrace(err))
 			}
 		}
+
+		ipv6EgressRule := sg.getDefaultSecurityGroupIPv6EgressRule()
+		_, err = sg.svc.RevokeSecurityGroupEgress(ipv6EgressRule)
+		if err != nil {
+			if awsErr, isAwsErr := err.(awserr.Error); isAwsErr && awsErr.Code() == "InvalidPermission.NotFound" {
```
Comment (denis256) [inline aws/ec2.go:750]: Error handling seems to be similar to the handling of `ingressRule / egressRule` errors , was wondering if will be helpful if handling can be extracted to a common function?

**Review by denis256 — APPROVED** [review]

File: aws/ec2.go (hunk 1 above)
Comment (marinalimeira) [reply aws/ec2.go:750]: Yes, good point. I will leave as it is since it's outside the scope of this PR, and open an issue! Thanks for the review

**Comment by marinalimeira** [comment]
Thank you everybody for the review!! 🎉

**Pull request merged**